### PR TITLE
SASS-4183

### PR DESCRIPTION
**Pull request by srinivaspach**
# Checklist

Jira subtask: SASS-4247

## Developer: Before Asking Others To Review Your Code

Have you ...

- [X]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [X]  Hosted a group review of your implementation?
- [X]  Rebased against `main`? 
- [X]  Ensured that your commit(s) contain the ticket key?
- [X]  Ensured that your branch is named after the ticket key?
- [X]  Successfully run the unit tests?
- [X]  Successfully run the integration tests?
- [X]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [ ]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [ ]  Pulled the branch down?
- [ ]  Assigned yourself to this PR?
- [ ]  Moved the Jira *subtask* to “in review”?

## Developer: After Approval

Have you ...

- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

File: app/forms/Countries.scala [hunk 1]
```diff
@@ -31,13 +31,28 @@ object Countries {
       .map(_.replace("\"", ""))
       .toList
 
+  val countriesThreeDigitMap : Map[String, String] = {
+    val filename = "conf/three-digit-country-code-to-country-name.csv"
```
Comment (abiodxn) [inline app/forms/Countries.scala:35]: I'm not sure this file should be in the conf file folder. seems more apt to be in the resources package

File: app/forms/Countries.scala [hunk 2]
```diff
@@ -31,13 +31,28 @@ object Countries {
...
+    val lines = Source.fromFile(filename).getLines.toList
+    lines.foldLeft(List[(String, String)]()) {
+      (acc, item) =>
+        val lineArray = item.split("=")
+        val (countryCode, countryName) = (lineArray.head, lineArray.last)
```
Comment (abiodxn) [inline app/forms/Countries.scala:40]: .head and .last could throw exception here if the list is empty (very unlikely but i think we should handle it just incase)

Comment (srinivaspach) [reply app/forms/Countries.scala]: Fixed

File: app/forms/Countries.scala (hunk 1 above)
Comment (srinivaspach) [reply app/forms/Countries.scala:35]: resources has not been used in any of our repos

**Review by abiodxn — APPROVED** [review]

**Pull request merged**